internal/server/iigointernal/orchestration.go

```diff
@@ -228,6 +228,8 @@ func RunIIGO(logger shared.Logger, g *gamestate.GameState, clientMap *map[shared
 	var appointJudgeError, appointSpeakerError, appointPresidentError error
 	actionCost := gameConf.IIGOConfig
 	costOfElection := actionCost.AppointNextSpeakerActionCost + actionCost.AppointNextJudgeActionCost + actionCost.AppointNextPresidentActionCost
+	//reset IIGO Election before using it
+	g.IIGOElection = nil
```

Comment:
I know obv we use the append function so it takes care of the nil, but could we maybe just instantiate to empty or smth, just in case we want to check this down the line and it becomes nil dereference.

Reply:
`g.IIGOElection = make([]gamestate.VotingInfo, 0)`

did this instead hopefully thats alright

Reply:
Yup, looks good